### ISSUE TITLE
build failures

### ISSUE THREAD

**Issue by graphaelli** [elastic/apm-integration-testing#176]
Lots of build failures lately:

```
17:18:44 waiting for services to be healthy
17:18:44 docker-compose-wait
17:18:45 Some processes failed
17:18:45 Makefile:72: recipe for target 'dockerized-test' failed
```

apm-server is not coming up properly:
```
localtesting_7.0.0-alpha1_apm-server | {"level":"info","timestamp":"2018-09-11T21:39:22.464Z","caller":"instance/beat.go:373","message":"apm-server stopped."}
localtesting_7.0.0-alpha1_apm-server | {"level":"error","timestamp":"2018-09-11T21:39:22.464Z","caller":"instance/beat.go:757","message":"Exiting: Error importing Kibana dashboards: fail to create the Kibana loader: Error creating Kibana client: Error creating Kibana client: fail to get the Kibana version: HTTP GET request to /api/status fails: parsing kibana response: invalid character 'K' looking for beginning of value. Response: Kibana server is not ready yet."}
localtesting_7.0.0-alpha1_apm-server | Exiting: Error importing Kibana dashboards: fail to create the Kibana loader: Error creating Kibana client: Error creating Kibana client: fail to get the Kibana version: HTTP GET request to /api/status fails: parsing kibana response: invalid character 'K' looking for beginning of value. Response: Kibana server is not ready yet.
```

The kibana healthcheck isn't holding apm-server back long enough for kibana to really be available.

**Comment by graphaelli** [comment]
Ideally the kibana healthcheck would be better.  In the meantime, #177 is something that should be done anyway and may reduce the urgency on this bug.

**Comment by graphaelli** [comment]
closed by #178
